worksheets/03-local-storage.md

```diff
@@ -58,7 +58,7 @@ After running these three lines `name` will be set to Susan.
 
 In the last worksheet you explored [forms and events](02-forms-and-events.md).
 
-1. Make an text input asks the user's name, stores it to Local Storage
+1. Make a text input that asks the user's name, then stores it to Local Storage
 2. Add a button which deletes the `name` from Local Storage
 
-Once you've done those, move onto [Arrays and Objects](04-arrays-objects.md).
\ No newline at end of file
+Once you've done those, move onto [Arrays and Objects](04-arrays-objects.md).
```

Comment:
I can't work out why this line has changed :P

Reply:
Think there wasn’t a beeline at the end and now there is.